docs/utils/component.config.json

```diff
@@ -683,7 +683,7 @@
   {
     "id": "use-media-query",
     "name": "useMediaQuery",
-    "title": "",
+    "title": "不稳定 🧪",
```

Comment:
`title` sets the title of `hook`, not the translation of `tag`. 

![image](https://github.com/rsuite/rsuite/assets/1203827/39aca9e0-4e81-42f6-8fa6-c282cb4552da)


> https://rsuite-nextjs-git-fork-deardreamweb-main-rsuite.vercel.app/zh/components/use-media-query/